service/kvmclusterconfig/v1/key/key.go

```diff
@@ -1,10 +1,15 @@
 package key
 
 import (
+	"fmt"
+
 	"github.com/giantswarm/apiextensions/pkg/apis/core/v1alpha1"
 	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/randomkeytpr"
 )
 
+// ToCustomObject converts value to v1alpha1.KVMClusterConfig and returns it or
+// error if type does not match.
 func ToCustomObject(v interface{}) (v1alpha1.KVMClusterConfig, error) {
```

Comment:
I would like to see the functions being sorted alphabetically like in the other projects.